bip-p2qrh.mediawiki

```diff
@@ -209,8 +213,8 @@ To help implementors understand updates to this BIP, we keep a list of substanti
 
 * 2024-06: High level rough draft
 * 2024-07: Additional algorithms in PQC table
-* 2024-08: Add FALCON signatures, update to use NIST standard terminology, add public key sizes
-* 2024-09: Additional detail on P2QS
+* 2024-08: Add FALCON signatures, update to use NIST standard terminology, add public key sizes.
+* 2024-09: Additional detail on P2QS. Deprecate P2QR. Postpone SQIsign.
```

Comment:
Do we need to make clear that we are deprecating P2QR in favor of having only P2QRH?  I mean, we do if you read the BIP, but at a high level glance you might not know.  And I guess that would be the reader's loss.